[2025-04-18T12:07:43.738+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T12:07:43.777+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_gcs.transform_csv scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2025-04-18T12:07:43.804+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_gcs.transform_csv scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2025-04-18T12:07:43.806+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-18T12:07:43.927+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform_csv> on 2004-01-01 00:00:00+00:00
[2025-04-18T12:07:43.962+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2504) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-18T12:07:43.966+0000] {standard_task_runner.py:72} INFO - Started process 2507 to run task
[2025-04-18T12:07:43.958+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_gcs', 'transform_csv', 'scheduled__2004-01-01T00:00:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_to_gcs.py', '--cfg-path', '/tmp/tmp9i3b3h4e']
[2025-04-18T12:07:43.969+0000] {standard_task_runner.py:105} INFO - Job 568: Subtask transform_csv
[2025-04-18T12:07:44.101+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_to_gcs.transform_csv scheduled__2004-01-01T00:00:00+00:00 [running]> on host f21ba0bbe878
[2025-04-18T12:07:44.395+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_gcs' AIRFLOW_CTX_TASK_ID='transform_csv' AIRFLOW_CTX_EXECUTION_DATE='2004-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2004-01-01T00:00:00+00:00'
[2025-04-18T12:07:44.398+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T12:08:29.907+0000] {job.py:229} INFO - Heartbeat recovered after 15.42 seconds
[2025-04-18T12:08:38.050+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-04-18T12:08:38.316+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T12:08:38.320+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
